src/components/Icons/discord.tsx

```diff
@@ -1,7 +1,11 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-function IconDiscord({ className = '' }) {
+type Props = {
+  className?: string;
+};
+
+function IconDiscord({ className }: Props) {
```

Comment:
I'm fine with leaving it the way it is, but I think we'd also be fine to establish a pattern of inlining non-exported prop types.

```suggestion
function IconDiscord({ className }: {className?: string}) {
```